android/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="persona_inquiry_start_title">Valora would like to verify your identity</string>
+  <string name="persona_inquiry_start_title">Mobile Stack would like to verify your identity</string>
```

Comment:
```suggestion
  <string name="persona_inquiry_start_title">Verify your identity</string>
```

I think this string is user facing and "Mobile Stack" might be confusing..so trying to think of app name agnostic strings. other options i can think of are "this app would like..." or "we would like..."?